src/components/SearchResults.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { 
+    Button,
+    Card, 
+    CardContent, 
+    CardMedia,
+    Grid, 
+    Typography 
+} from '@material-ui/core'; 
+
+
+const styles = theme => ({
```

Comment:
do you use theme variable? as eslint is outputting errors, you should delete it if you don't use.